tests/components/recorder/test_util.py

```diff
@@ -60,3 +62,47 @@ def to_native(validate_entity_id=True):
         util.execute((mck1,), to_native=True)
 
     assert e_mock.call_count == 2
+
+
+async def test_validate_or_move_away_sqlite_database(hass, tmpdir, caplog):
```

Comment:
Let's drop `async` and we don't have to run anything in the executor.

Reply:
good idea. should be slightly faster.

done